src/scenario.cpp

```diff
@@ -306,4 +321,12 @@ const std::string &scenario::get_map_special() const
 {
     return _map_special;
 }
+std::vector<mission_type_id> scenario::missions() const
+{
+    std::vector<mission_type_id> res;
+    for( const mission_type_id &m : _missions ) {
+        res.emplace_back( m );
```

Comment:
What's the point, was `_missions` of a different type before? You can simply return a constant reference to it.

Reply:
Was an artifact of some filtering code I had in here to return a subset of the missions while experimenting with it. Changed it to a const reference.